plugins/generator-1.18.2/forge-1.18.2/templates/elementinits/items.java.ftl

```diff
@@ -61,9 +61,9 @@ public class ${JavaModName}Items {
             public static final RegistryObject<Item> ${item.getModElement().getRegistryNameUpper()}_BOOTS =
                 REGISTRY.register("${item.getModElement().getRegistryName()}_boots", () -> new ${item.getModElement().getName()}Item.Boots());
             </#if>
-        <#elseif item.getModElement().getTypeString() == "dimension">
-            public static final RegistryObject<Item> ${item.getModElement().getRegistryNameUpper()} =
-                REGISTRY.register("${item.getModElement().getRegistryName()}", () -> new ${item.getModElement().getName()}Item());
+        <#elseif item.getModElement().getTypeString() == "dimension" && item.enableIgniter>
+			public static final RegistryObject<Item> ${item.getModElement().getRegistryNameUpper()} =

```

Comment:
Keep indentation in those files at it was before, just add additional needed condition, applies to all versions

Reply:
Done!